src/parser/open-api/v3/navigationParser.js

```diff
@@ -56,5 +58,15 @@ export function getServicesMethod ({ path, method, request, params, responses })
     servicesMethod.parameters = params
   }
 
+  if (servers && servers.length > 0) {
+    servicesMethod.paths = servers.map(server => {
+      return {
+        url: server.url + path,
+        description: server.description
```

Comment:
It's not desired. I've updated the code to not include `description` when it doesn't exist.